CONTRIBUTING.md

```diff
@@ -1,55 +1,77 @@
-submitting a pull request!
+## Did you find a bug?
 
-## Commitlint
+### Before Submitting A Bug Report
 
-We use [Commitlint](https://github.com/conventional-changelog/commitlint/#what-is-commitlint) to help manage the semantic versions across the various packages in this library. Please be sure that you take this into concideration when submitting PRs to this repositiory. Generally, your commits should look like the following:
+-   **Check the [Issue Discussions](https://github.com/adobe/spectrum-web-components/issues).** You might be able to find the cause of the problem and fix things yourself. Most importantly, check if you can reproduce the problem [in the latest version of Spectrum Web Components](https://studio.webcomponents.dev/workspace/adobe)
 
-```
-type(scope?): subject #scope is optional, but should reference the package you are updating
-```
+-   **Perform a [search](https://github.com/adobe/spectrum-web-components/issues)** to see if the problem has already been reported. If it has **and the issue is still open**, add a comment to the existing issue instead of opening a new one.
 
-## From Contributor To Committer
+### How Do I Submit A (Good) Bug Report?
 
-We love contributions from our community! If you'd like to go a step beyond contributor
-and become a committer with full write access and a say in the project, you must
-be invited to the project. The existing committers employ an internal nomination
-process that must reach lazy consensus (silence is approval) before invitations
-are issued. If you feel you are qualified and want to get more deeply involved,
-feel free to reach out to existing committers to have a conversation about that.
+Bugs are tracked as [GitHub issues](https://guides.github.com/features/issues/). Create an issue on the repository and provide the following information by filling in [the template](https://github.com/adobe/spectrum-web-components/blob/main/.github/ISSUE_TEMPLATE/bug_report.yaml).
 
-## Security Issues
+[IMPORTANT] Explain the problem and include additional details to help maintainers reproduce the problem:
+
+-   **Describe the exact steps which reproduce the problem** in as many details as possible.
+-   **Provide specific examples to demonstrate the steps**. If you're providing snippets in the issue, use [Playground](https://studio.webcomponents.dev/workspace/adobe).
+-   **Explain which behavior you expected to see instead and why.**
+-   **Include screenshots and animated GIFs** which show you following the described steps and clearly demonstrate the problem.
+
+## Do you intend to add a new feature or change an existing one?
+
+-   Suggest your change in the swc-core mailing list and start writing code.
+
+-   Do not open an issue on GitHub until you have collected positive feedback about the change. You can start on [Adding your idea here](https://github.com/adobe/spectrum-web-components/discussions/categories/ideas) under Ideas
```

Comment:
I think it's fine if people want to request features through our issue template, but it probably makes our issues more organised if we keep those to the discussions. 🤔